schema/tables/system_state.yml

```diff
@@ -0,0 +1,18 @@
+name: system_state
```

Comment:
We'll want to add a note to indicate that this is not a core osquery table - see [discussion](https://fleetdm.slack.com/archives/C01EZVBHFHU/p1687467006285049?thread_ts=1687465572.630919&cid=C01EZVBHFHU)

Reply:
100, I'm going to copy and paste @sharon-fdm 's note for consistency

Reply:
I copied and pasted @jacobshandling 's merged note for consistency!